src/main/resources/default/templates/biz/login-form.html.pasta

```diff
@@ -13,7 +13,7 @@
                name="user"
                type="text"
                autofocus="autofocus"
-               value="@call.get('user').asString()"
+               value="@CallContext.getCurrent().get(WebContext.class).get('user').asString()"
```

Comment:
A macro would be nice for this

Reply:
yep, I'd still put that in another PR (a ScopeConfig macro would be nice as well) - still lets keep this concise..